source/includes/mongodb-compatibility-table-go.rst

```diff
@@ -23,7 +23,7 @@
      - ✓
      - ✓
      - ✓
-     -
+     - ✗
```

Comment:
S: consider asking if this is the case that connecting to 3.6 causes an unfixable error or if its just not tested.

Reply:
@jtazin Can you provide some context on which mark we should put here?